### PR TITLE
skip tests - push images to GCR

### PR DESCRIPTION
**Pull request by jacmrob**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by jacmrob** [comment]
tested by running a build off UI - https://fc-jenkins.dsp-techops.broadinstitute.org/view/Build/job/firecloud-ui-build/4000/
and by running a gcr push build off UI - https://fc-jenkins.dsp-techops.broadinstitute.org/view/Build/job/firecloud-ui-build-2/6/

DSL changes to change build command forthcoming.

File: jenkins/jenkins_build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eux
+
+GCR_SVCACCT_VAULT="secret/dsde/dsp-techops/common/dspci-wb-gcr-service-account.json"
```
Comment (jmthibault79) [inline jenkins/jenkins_build.sh:5]: This filename is used several times - let's make it a variable

**Review by jmthibault79 — APPROVED** [review]

File: scripts/build.sh [hunk 2]
```diff
@@ -11,6 +11,93 @@
...
+ENV=${ENV:-""}
+SERVICE_ACCT_KEY_FILE=""
+
+BUILD_CLJ=false
+RUN_DOCKER=false
+PRINT_HELP=false
+
+if [ -z "$1" ]; then
+    echo "No argument supplied!"
+    echo "run '${0} -h' to see available arguments."
```
Comment (aweng98) [inline scripts/build.sh:51]: when I typed `./build.sh -h`, help information outputs contained formatting and non-printing char, and some scripts.

**Comment by aweng98** [comment]
build then compile the image without push doesn't work. Is it supported?
`./docker/build.sh compile -d build` yields
`invalid argument "broadinstitute/:da60da1e1824" for "-t, --tag" flag: invalid reference format`

File: scripts/build.sh [hunk 3]
```diff
@@ -11,6 +11,93 @@
...
+   Jenkins build job should run with all options, for example,
+     ./docker/build.sh compile -d push
+   To build the compile, the image, and push it to a gcr repository.
+     ./docker/build.sh compile -d build -r gcr --project "my-awesome-project"
+\t
+EOF
+)"
+
+# Enable strict evaluation semantics
+set -e
```
Comment (aweng98) [inline scripts/build.sh:34]: nit. `set` can use `-u` exits shell with an error when reference an undefined variable, like in `jenkins_build.sh`

File: scripts/build.sh [hunk 4]
```diff
@@ -11,6 +11,93 @@
...
+
+# Set default variables
+DOCKER_CMD=
+BRANCH=${BRANCH:-$(git rev-parse --abbrev-ref HEAD)}  # default to current branch
+DOCKERHUB_REGISTRY=${DOCKERHUB_REGISTRY:-broadinstitute/$PROJECT}
```
Comment (aweng98) [inline scripts/build.sh:39]: I'm new to this so it's not clear to me as to what the default value is for variable `DOCKERHUB_REGISTRY`?

**Review by aweng98 — APPROVED** [review]

**Pull request merged**